utils/saveData.js

```diff
@@ -1,9 +1,23 @@
 export const windowGlobal = typeof window !== 'undefined' && window;
 
-export const saveData = (formPath, data, step) => {
+export const saveData = (
+  formPath,
+  data,
+  title,
+  step,
+  includesDetails,
```

Comment:
Minor thing but I think we need to deconstruct all these arguments, we can get rid of a lot of duplication 🤔